code/__DEFINES/dcs/signals/signals_misc.dm

```diff
@@ -0,0 +1,121 @@
+/// Sent from /obj/structure/industrial_lift/tram when its travelling status updates. (travelling)
```

Comment:
We definitely should not have a misc file, people are just going to throw all their stuff in there.

Most of these should be split into their own files.